### ISSUE TITLE
800x600 widescreen not working (1068x600)

### ISSUE THREAD

**Issue by oGkspAz** [mir-diablo-ii-tools/SlashGaming-Diablo-II-Free-Resolution#78]
I can get 800x600 and 1280x720 to work but it refuses to use 1066x600 (tried 1067 and 1068 too)
D2DX handles this fine but it currently has compatibility issues with d2lootfilter.

**Comment by oGkspAz** [comment]
I thought it might be me using the default glide or something so closed issue when I was at work today. Just tested it with sven's glide wrapper (1.4e) and it refuses to run at 1068x600.
SGD2FreeResolution.json looks as follows:

> {
    "SlashGaming Diablo II Free Resolution": {
        "!!!Metadata (Do not modify)!!!": {
            "Major Version A": 3,
            "Major Version B": 0,
            "Minor Version A": 1,
            "Minor Version B": 0
        },
        "Ingame Resolutions": [
            "640x480",
            "800x600",
            "1068x600"
        ],
        "Ingame Resolution Mode": 0,
        "Main Menu Resolution": "800x600",
        "Custom MPQ File": "SGD2FreeRes.mpq",
        "Enable Screen Border Frame?": true,
        "Use Original Screen Border Frame?": false,
        "Use 800 Interface Bar?": true
    },
    "!!!Globals!!!": {
        "Config Tab Width": 4
    }
}

**Comment by master861** [comment]
you mean you get SGD2FreeResolution running? i not even get it running only from balrogs dll but not this dll her.

**Comment by oGkspAz** [comment]
It does run yes. In my case, because I am using BaseMod, I load it with that:

> [ExtraDll1]
> Enabled=1
> LoadDll1="SGD2FreeRes.dll"
> 
> [ExtraDll2]
> Enabled=1
> LoadDll2="d2lootfilter.dll"
> 
> [ExtraDll3]
> Enabled=1
> LoadDll3="PlugY.dll"
> 
> ; ExtraDll allows you to load other dlls in the order above, up to 31 char in length.

**Comment by oGkspAz** [comment]
Adding -3dfx allows 1068x600 to work but with black borders top and bottom as well as on both sides.

**Comment by oGkspAz** [comment]
TLDR: run with -3dfx and open up Sven's glide-ini.exe, select settings and tick desktopresolution.
